src/libtomahawk-playdarapi/Api_v1_5.cpp

```diff
@@ -105,7 +105,7 @@ Api_v1_5::playback( QxtWebRequestEvent* event, const QString& command )
             } else {
                 trackInfo.insert( "resolvedBy", "<unknown resolver>" );
             }
-            trackInfo.insert( "score", currentTrack->score() );
+//FIXME?            trackInfo.insert( "score", currentTrack->score() );
```

Comment:
Keep this for internal weighting?

A resolver might know that certain hosts are less reliable than others..